### DISCUSSION TITLE
Porte "aperte" che lampeggiano sulla planimetria. Come?

### DISCUSSION THREAD

**Discussion by Jones1982** [indomus/forum#286] [HUB: Home Assistant]
Buona sera a tutti!
C'è qualche integrazione, codice o altro, che mi possa fare "pulsare/lampeggiare" le icone dei sensori porta, sulla mappa qui sotto allegata?

![Cattura](https://user-images.githubusercontent.com/57654443/117548471-7fdedf80-b035-11eb-82c7-045348303117.PNG)

Di base, cambiano colore in giallo (quando il contatto porta segnala l'apertura), ma è poco visibile se non si è vicino al monitor.

Consigli?
Grazie sempre!!!

**Comment by Jones1982** [comment]
Ah, un altra cosa:
i valori dei sensori t&u, si possono ruotare di 90°? 
Cioè invece di vederli in orizzontale, vorrei ruotare tutta la sezione di 90° x collegarla al tramezzo.

**Comment by dante70** [comment]
**LINK RIMOSSO**
io son riuscito a fare qualcosa seguendo questa guida

**Reply by snakuzzo** [reply]
Non sono ammessi link ad altri siti...cancello.
Leggete il regolamento!
